web/packages/teleterm/src/agentCleanupDaemon/agentCleanupDaemon.js

```diff
@@ -59,13 +68,36 @@ if (!process.send) {
   );
 }
 
-const logLine = `parent=${parentPid} agent=${agentPid} self=${process.pid} cluster=${rootClusterUri}`;
+const logger = createLogger({
+  level: 'info',
+  exitOnError: false,
+  format: format.combine(
+    format.timestamp({
+      format: 'DD-MM-YY HH:mm:ss',
+    }),
+    format.simple()
+  ),
```

Comment:
This doesn't quite follow the format of our other logs, but I think that's fine.